internal/core/metadata/rest_deviceprofile_test.go

```diff
@@ -109,12 +109,10 @@ func TestGetAllProfiles(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			LoggingClient = logger.MockLogger{}
 			Configuration = &ConfigurationStruct{Service: config.ServiceInfo{MaxResultCount: len(TestDeviceProfiles)}}
 			dbClient = tt.dbMock
 			rr := httptest.NewRecorder()
-			handler := http.HandlerFunc(restGetAllDeviceProfiles)
-			handler.ServeHTTP(rr, tt.request)
```

Comment:
@michaelestrin Similarly, as we're not using `tt.request` anymore, should it be refactored out now?

Reply:
I concur.  Since we're not using it anywhere, `request` should be removed from the anonymous struct assigned to the `tests` variable.